### ISSUE TITLE
img alt

### ISSUE THREAD

**Issue by ghost** [hugovanversendaal/codecademy-teacozy#4]
https://github.com/hugovanversendaal/codecademy-teacozy/blob/c25dc942953aefe7de69bc340495fc03ddf668c1/index.html#L35-L59

Nice job thoroughly giving all  of your `img` tags a descriptive `alt` attribute, just in case your website is not able to load it! 👍 

Doing so will help improve the accessibility of your website when it's visited by visually impaired users on screen readers-- as well as any other time that your website is just unable to render images.
